t/tasks/inventory/windows/license.t

```diff
@@ -118,7 +118,7 @@ ok( @licenses == 1 );
 ok( $licenses[0]->{'KEY'} eq 'YKP6Y-3MDM7-J8F3Q-9297J-3TF27' );
 ok( $licenses[0]->{'PRODUCTID'} eq '00339-10000-00000-AA310' );
 
-push @licenses, FusionInventory::Agent::Task::Inventory::Win32::License::_getWmiSoftwareLicensingProducts();
+@licenses = FusionInventory::Agent::Task::Inventory::Win32::License::_getWmiSoftwareLicensingProducts(@licenses);
 
 # License was still read from registry, no license added
 ok( @licenses == 1 );
```

Comment:
It would nice here to add a test case which is failing before you fix and working after.